homework-submission/myapp/src/Components/Data.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Todos from './Todos.json';
+
+class Data extends React.Component {
+
+    state = {
+        Todos
+    }
+
+    isTaskDone = (todo) => {
+        todo.done = !todo.done;
+        this.setState({
+            Todos
+        });
+    }
+
+    renderTodos() {
+        if (this.state.Todos.length === 0) return <p className='noItems'>No items...</p>
+
+        const textStyle = { textDecoration: 'line-through', backgroundColor: '#0455d8' }
+
+        return <ul>
+            {
+                Todos.map((todo, id) => {
+
+                    return (
+                        <li key={id}>
+                            <input checked={todo.done} type='checkbox' onChange={this.isTaskDone.bind(this, todo)} />
+                            <span style={todo.done ? textStyle : null}>
```

Comment:
line 28, you don't need to bind since you are already using arrow functions

try this: 
```javascript
onChange={() => this.isTaskDone(todo)}
````